### ISSUE TITLE
timeout errors when migrating

### ISSUE THREAD

**Issue by lsat12357** [OregonDigital/OD2#862]
After ingesting several dozen assets on dev, the following error starts affecting the admin_set_membership_service:
fx719w509 did not obtain sets: Net::ReadTimeout : ["/usr/local/lib/ruby/2.5.0/net/protocol.rb:181:in `rbuf_fill'"
In addition, the collections/all display in the front end also errors:
Faraday::TimeoutError in Hyrax::Dashboard::Collections#index
It can be fixed by restarting the containers.

**Comment by jechols** [comment]
I did a lot of investigation here, but didn't conclude much.  I was able to reproduce the problem, but it's more intermittent for me than it was for @lsat12357.

Facts (or ... things I think are facts maybe):

- It seems that the problem is more likely to occur when a lot of workers are running.  For instance, when you spin up an extra `hyrax_migrator` sidekiq process.
- I logged *all* Faraday traffic, and the only things it hits are Solr and Fedora.  @lsat12357 said her stack trace didn't have any RSolr in it, and I'm positive I didn't notice that in mine, either (but being intermittent I no longer have a stack trace to look at).  This suggests Fedora is having problems in some way.
- Honeycomb is in the stack trace, but this is a red herring: honeycomb uses a "NullClient" or something, so it's going to show up in pretty much every stack trace even when it's not actually doing anything.  Hooray for magic code injection :roll_eyes: 
- I believe I saw somewhere that the timeout was 5000ms, so I'm very inclined to believe that something just slows Fedora down a lot.  5 seconds isn't a lot for a heavy ingest into a dev instance of OD2.  I mean, it's a lot for a stack that's not built on a foundation of garbage, but our stack is known to be dreadfully slowwwwww.  So if Faraday has a 5000ms timeout by default, we may need to try upping that in dev.  Of course that may just mask a bigger problem.  Hard to say.
